docs/meta/terraform.md.tmpl

```diff
@@ -1,19 +1,5 @@
 <!--- This file is autogenerated, edit the template in meta/ -->
```

Comment:
Instead of modifying this template, we should just stop auto-generating the `terraform.md` file.

Reply:
The generator currently does not auto run but has to be manually triggered. So this file should not be updated unless manually done by running the code.

I can remove that code although I thought leaving the code was ok for now.

Reply:
@kaariger how are we generating https://g.co/dev/terraformfoundation? I assumed that was using this template?

Reply:
> @kaariger how are we generating https://g.co/dev/terraformfoundation? I assumed that was using this template?

That is currently being manually maintained.

Reply:
Looking at the source of https://g.co/dev/terraformfoundation it does look like the generated txt is slightly different but seems like something we should add to the template.